R/utils.R

```diff
@@ -448,3 +448,7 @@ names_to_lazy_tbl <- function(x) {
   frame <- as.data.frame(stats::setNames(rep(list(logical()), length(x)), x))
   dbplyr::tbl_lazy(frame)
 }
+
+is_list_empty <- function(x) {
```

Comment:
This _could_ just be `is_empty` as it will detect empty vectors, matrices, data.frame etc as well.